### PR TITLE
Edit DeleteCommand

### PR DESCRIPTION
**Pull request by asaierika**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -33,15 +37,40 @@ public DeleteCommand(Index targetIndex) {
...
+                model.deletePerson(studentToDelete);
+                entityInformation = studentToDelete.toString();
+                break;
+            case TUTOR_LIST:
+                lastShownTutorList = model.getFilteredTutorList();
+                Tutor tutorToDelete = lastShownTutorList.get(targetIndex.getZeroBased());
+                model.deletePerson(tutorToDelete);
+                entityInformation = tutorToDelete.toString();
+                break;
+            default:
```
Comment (tantzeyeong) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:61]: Would it be better to have a case for TUITIONCLASS_LIST and let the default throw an exception?

**Review by tantzeyeong — COMMENTED** [review]
Just some minor changes to consider, such as the error message formats given with invalid inputs. Other than that, LGTM!

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java (hunk 1 above)
Comment (asaierika) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java:61]: Thats actually what I did last time maybe can discuss which is better later

**Comment by tantzeyeong** [comment]
LGTM!

**Pull request merged**